src/domain/alerts/alerts.repository.ts

```diff
@@ -7,12 +7,17 @@ import { AlertLog } from '@/routes/alerts/entities/alert.dto.entity';
 import { DelayModifierDecoder } from '@/domain/alerts/contracts/delay-modifier-decoder.helper';
 import { SafeDecoder } from '@/domain/alerts/contracts/safe-decoder.helper';
 import { MultiSendDecoder } from '@/domain/alerts/contracts/multi-send-decoder.helper';
+import { IEmailRepository } from '@/domain/email/email.repository.interface';
 
 @Injectable()
 export class AlertsRepository implements IAlertsRepository {
   constructor(
     @Inject(IAlertsApi)
     private readonly alertsApi: IAlertsApi,
+    // TODO: IEmailRepository needs to be injected to be used in
+    // https://github.com/safe-global/safe-client-gateway/pull/872
+    @Inject(IEmailRepository)
+    private readonly emailRepository: IEmailRepository,
```

Comment:
I've left this injection here just for visibility and for demonstration purposes, as the main goal of the PR is to make this injection possible, but in this commit the injected class is not used. It will be used by [the referenced PR](https://github.com/safe-global/safe-client-gateway/pull/872).

Reply:
This change was removed. It will be done in a separate PR.